docs/site/angular-auth-oidc-client/docs/documentation/auto-login.md

```diff
@@ -5,102 +5,102 @@ sidebar_position: 9
 
 # Auto Login
 
-The library supports route-based automatic login thanks to the route guard: `AutoLoginPartialRoutesGuard`. The guard implements the necessary handlers for both `canActivate` and `canLoad`, and will preserve the route upon completing a successful login.
-
-:::info
-
-If you are using multiple configurations, the guards currently select the first provided configuration to perform the login.\_
-
-:::
+The library supports route-based automatic login thanks to the functional route guard: `autoLoginPartialRoutesGuard`. The guard implements the necessary handlers for both `canActivate` and `canMatch`, and will preserve the route upon completing a successful login.
 
 ## Common Scenarios
 
 Here are a couple of the common use cases.
 
-### Auto Login when default route is not guarded
+### Auto Login when the default route is not guarded
 
-In this use case, some of your routes should be freely accessible, while others should be protected by a login, and the login should start when the user enters the protected route.
+In this use case, some of your routes should be freely accessible, while others should be protected by a login, which should start when the user enters the protected route.
 
 ```ts
-import { AutoLoginPartialRoutesGuard } from 'angular-auth-oidc-client';
+import { autoLoginPartialRoutesGuard } from 'angular-auth-oidc-client';
 
 const routes: Routes = [
   { path: '', pathMatch: 'full', redirectTo: 'home' },
   { path: 'home', component: HomeComponent },
   {
     path: 'protected',
     component: ProtectedComponent,
-    canActivate: [AutoLoginPartialRoutesGuard],
+    canActivate: [autoLoginPartialRoutesGuard],
   },
   {
     path: 'customers',
-    loadChildren: () =>
-      import('./customers/customers.module').then((m) => m.CustomersModule),
-    canLoad: [AutoLoginPartialRoutesGuard],
+    loadChildren: () => import('./customers/customers.routes').then((m) => m.customerRoutes),
+    canLoad: [autoLoginPartialRoutesGuard],
   },
   { path: 'unauthorized', component: UnauthorizedComponent },
 ];
 ```
 
 In this case, the `/home` and `/unauthorized` routes are not protected, and they are accessible without a login.
-Please make sure to call `checkAuth()` like normal in your `app.component.ts`.
-
-```ts
-export class AppComponent implements OnInit {
```

Comment:
I removed this because we also have the `withAppInitializerAuthCheck`, and I didn't want to bloath these docs with adding an example with `withAppInitializerAuthCheck`.